docs/faq/security.mdx

```diff
@@ -61,7 +61,7 @@ A `Multi-Sig wallet` is a smart contract wallet that provides security by only a
 ## The fake NFT mint
 One of the most common scams I come across is the Fake NFT mint website, a site that's set up to look like an NFT project where you connect your wallet, press a button to mint, and approve a transaction to "get your NFT".
 
-<i>Except,</i> what the website does is spawn a transaction that either directly transfers one of your NFT's out or adds a `Token Approval` which gives another wallet full access to one of your NFT collections.
+*Except,* what the website does is spawn a transaction that either directly transfers one of your NFT's out or adds a `Token Approval` which gives another wallet full access to one of your NFT collections.
```

Comment:
Did this fix the `code` syntax not displaying properly? Removing the `<i>` was just a guess, something about having those two on the same line confuses the interpreter maybe.

Reply:
> Did this fix the `code` syntax not displaying properly? Removing the `<i>` was just a guess, something about having those two on the same line confuses the interpreter maybe.

Yep! The interpreter easily gets confused, not just by things on the same line but also sometimes on adjacent lines.